code/game/objects/items/tools/crowbar.dm

```diff
@@ -162,3 +162,53 @@
 	usesound = 'sound/items/jaws_pry.ogg'
 	force = 10
 	toolspeed = 0.5
+
+/obj/item/crowbar/mechremoval
+	name = "mech removal tool"
+	desc = "A... really big crowbar. You're pretty sure it could pry open a mech, but it seems unwieldy otherwise."
+	icon_state = "mechremoval0"
+	base_icon_state = "mechremoval"
+	inhand_icon_state = null
+	icon = 'icons/obj/mechremoval.dmi'
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = NONE
+	toolspeed = 1.25
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 100, BIO = 0, FIRE = 100, ACID = 0)
+	resistance_flags = FIRE_PROOF
+	bare_wound_bonus = 15
+	wound_bonus = 10
+	/// How much damage to do unwielded
+	var/force_unwielded = 5
+	/// How much damage to do wielded
+	var/force_wielded = 19
+
+/obj/item/crowbar/mechremoval/Initialize(mapload)
+	. = ..()
+	transform = transform.Translate(0, -8)
```

Comment:
why pixel shift like this?

Reply:
pixel changes dont carry over to huds

Reply:
this doesnt carry over to attack anims though but i dont think pixel changes do either